src/renderers/dom/ReactDOMServerBrowserEntry.js

```diff
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @providesModule ReactDOMServerEntry
+ * @providesModule ReactDOMServerBrowserEntry
  */
 
```

Comment:
We could add shims that throw for `renderToStream()` APIs here. Then it's less mysterious.